docker/runner.Dockerfile

```diff
@@ -28,3 +28,7 @@ COPY docker/.artifacts/bundle.car $FM_HOME_DIR/bundle.car
 COPY docker/.artifacts/contracts $FM_HOME_DIR/contracts
 COPY --from=builder /app/fendermint/app/config $FM_HOME_DIR/config
 COPY --from=builder /app/output/bin/fendermint /usr/local/bin/fendermint
+
+RUN useradd ipc && chown -R ipc $FM_HOME_DIR
+
+USER ipc
```

Comment:
Why do we need an `ipc` user instead of `root` here? Also, I don't think it is the case but I am wondering if this may break the execution of the `fendermint` container considering the directory structure that we use. Have you tried it?

Reply:
I never quite understood how to use users inside containers properly. I made images where I used `add-user` to create users with specific IDs and groups, but unless those IDs aligned with what exists on the host, I still had to end up setting the `-u` option to make sure I can remove files created into mounted volumes without `sudo`. Since this is a container, I never quite got why we shouldn't use `root` :shrug: I'm sure I'm just ignorant, but if it works...

Reply:
I removed as it's not needed anymore. But I think one day we should not use root to execute docker